almanac.lic

```diff
@@ -60,7 +60,7 @@ class Almanac
       DRC.bput("turn #{@almanac} to #{training_skill}", 'You turn', 'You attempt to turn')
     end
 
-    DRC.bput("study my #{@almanac}", 'You set about', 'gleaned all the insight you can', 'Study what', 'interrupt your research')
+    DRC.bput("study my #{@almanac}", 'You set about', 'gleaned all the insight you can', 'Study what', 'interrupt your research', 'The pages of the diamond-hide almanac seem worn')
```

Comment:
Note `diamond-hide` and `almanac` are the adjective and noun of the _almanac_. These can vary based on which ones have been purchased and/or alterations. Suggesting a wildcard in the regex to broaden the scope.
```suggestion
    DRC.bput("study my #{@almanac}", 'You set about', 'gleaned all the insight you can', 'Study what', 'interrupt your research', 'The pages of the .* seem worn')
```